lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr.cpp

```diff
@@ -29,13 +29,12 @@ extern "C" {
 }
 #endif
 
-#include "lte/gateway/c/core/oai/tasks/nas/esm/esm_ebr.hpp"
-
-#include "lte/gateway/c/core/oai/include/mme_app_ue_context.h"
+#include "lte/gateway/c/core/oai/include/mme_app_ue_context.hpp"
 #include "lte/gateway/c/core/oai/lib/3gpp/3gpp_24.007.h"
 #include "lte/gateway/c/core/oai/lib/bstr/bstrlib.h"
```

Comment:
General comment to move bstring file unser extern C and dynamic_memory_check.h file out of extern C